### PR TITLE
Little map additions and fixes.

### PR DESCRIPTION
**Pull request by yoyobatty**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: YoYoBatty
add: Containment pod doors in science now used unused hazard door sprite
add: Robotics vending machine
add: Toxins vending machine
add: Cameras in various areas 
add: Fire alarms in areas between firelocks so players don't get stuck without a crowbar
fix: Fixes circuit apc
fix: Fixes that annoying light inside the door below science
and many other little additions I missed in my previous PR's
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Camera in forge:

image

Camera in testing lab:

image

Alt pod door sprites:

image

image

Toxins vending machine (It was already in game):

image

Ditto for robotics vending machine:

image

Light was inside the door before:

image

Readds cameras to atmos gas tanks:

image

image

Camera at toxins launch access

Fixes random warning post on desk at vault:

image

### CODE REVIEW

**Comment by tortellinitony** [comment]
Pics please

File: _maps/map_files/Mining/Lavaland.dmm [hunk 1]
```diff
@@ -647,14 +647,17 @@
 	dir = 4
 	},
 /obj/structure/cable{
-	icon_state = "2-8"
+	icon_state = "4-8"
```
Comment (GuyonBroadway) [inline _maps/map_files/Mining/Lavaland.dmm]: Lavaland is still a tee-gee core file, if these are fixes for lavland could you do us a favour and move em upstream?

**Review by GuyonBroadway — DISMISSED** [review]

**Comment by Pyko1** [comment]
yeah, better try to push fixes to lavaland on tg, even tho it seems like you just flipped the mech charger around, but its bound to cause merge conflicts in future.

And id rather avoid merge issues instead of having minor fluff.

**Comment by JohnGinnane** [comment]
Looks good to me

**Review by JohnGinnane — APPROVED** [review]

**Pull request merged**